lib/docs/filters/html/clean_html.rb

```diff
@@ -6,6 +6,15 @@ def call
           node.before(node.children).remove
         end
 
+        if at_css('#browser_compatibility') and not at_css('#browser_compatibility').next_sibling.classes.include?('warning')
+          at_css('#browser_compatibility').next_sibling.remove
+          begin
+            compatibility_table = generateCompatibilityTable('html', slug, doc)
+            at_css('#browser_compatibility').add_next_sibling(compatibility_table)
+          rescue
+          end
```

Comment:
These are the debugging statements mentioned in the PR body, some pages still raise errors, I'm still trying to fix them.